library/pool.c

```diff
@@ -1242,8 +1272,11 @@ void dnet_io_stop(struct dnet_node *n) {
 	dnet_set_need_exit(n);
 
 	for (i = 0; i < io->net_thread_num; ++i) {
-		pthread_join(io->net[i].tid, NULL);
-		close(io->net[i].epoll_fd);
+		dnet_net_io_cleanup(&io->net[i]);
+	}
+
+	if (n->flags & DNET_CFG_JOIN_NETWORK) {
```

Comment:
Didn't find occurrence of this flag in "-" diff. Seems as something new. Is this flag unset only for client nodes?

Reply:
Yes, this flag is unset for all client nodes.